CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Changelog
 
+# [v1.1.0-rc.0](https://github.com/kubermatic/kubeone/releases/tag/v1.1.0-rc.0) - 2020-10-26
```

Comment:
Please update the release date before merging the PR.

Reply:
✅